build/release.sh

```diff
@@ -36,31 +38,25 @@ export BUILD_USER="$git_user"
 export BUILD_DATE=$( date +%Y%m%d ) # Release date is only to day-granularity
 export VERBOSE=true
 
-# Build the release binary with libpfm4 for docker container
-export GO_FLAGS="-tags=libpfm,netgo"
-build/build.sh
-
 # Build the docker image
 echo ">> building cadvisor docker image"
-gcr_tag="gcr.io/cadvisor/cadvisor:$VERSION"
-docker build -t $gcr_tag -f deploy/Dockerfile .
-
-# Build the release binary without libpfm4 to not require libpfm4 in runtime environment
-unset GO_FLAGS
-build/build.sh
-
-echo
-echo "double-check the version below:"
-echo "VERSION=$VERSION"
-echo
-echo "To push docker image to gcr:"
-echo "docker push $gcr_tag"
-echo
-echo "Release info (copy to the release page):"
-echo
-echo "Docker Image: N/A"
-echo "gcr.io Image: $gcr_tag"
```

Comment:
Let me know if I should build a similar release info page

Reply:
That would be great, to make it simpler to cut releases.

If we can just include the gcr tag, hash, and hashes of the cadvisor binaries, it's all we need

Reply:
Alright the latest version should print out a summary. I updated the releasing.md file to reflect the changes.